test-suite/djinni/all.djinni

```diff
@@ -1,4 +1,4 @@
 @import "common.djinni"
 
 @import "date.djinni"
```

Comment:
Not strictly related to this change, but it might be nice to have a test of a relative path which isn't based on an idl-include-path, e.g. by also moving date.djinni and including it as "vendor/third-party/date.djinni"